.github/workflows/ci.yaml

```diff
@@ -16,6 +16,11 @@ on:
 #
 # NOTE!!!
 
+env:
+  # etcd uses fsync by default, disable it for running on github actions
+  # xref: https://github.com/kcp-dev/kcp/pull/435/commits/064a517747d69c2cd8f7f8b4a595ad909e595c89
+  UNSAFE_MAC_E2E_HACK_DISABLE_ETCD_FSYNC: true
```

Comment:
why do we care for mac fsync hacks in CI ?

Side-note: I care a lot for it locally on my Mac ;-)

Reply:
or is that flag generic?

Reply:
It disables fsync on any platform afaik.

Reply:
Any platform. When I named it, I didn't think we'd need it elsewhere. Happy to rename